files/zh-cn/mozilla/add-ons/webextensions/api/webrequest/securityinfo/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: webRequest.SecurityInfo
+slug: Mozilla/Add-ons/WebExtensions/API/webRequest/SecurityInfo
+l10n:
+  sourceCommit: 702cd9e4d2834e13aea345943efc8d0c03d92ec9
+---
+
+{{AddonSidebar}}
+
+描述特定 Web 请求安全属性的对象。此类型的对象由 {{WebExtAPIRef("webRequest.getSecurityInfo()")}} API 返回。
+
+如果请求未使用 [TLS](/zh-CN/docs/Glossary/TLS) 加密，则此对象将只包含取值为“insecure”的 `state` 属性。
+
+## 类型
+
+此类型的值是对象，包含以下属性：
+
+- `certificates`
+
+  - : {{WebExtAPIRef("webRequest.CertificateInfo", "CertificateInfo")}} 的数组（`Array`）。如果调用 {{WebExtAPIRef("webRequest.getSecurityInfo()")}} 时传入了值为 `true` 的 `certificateChain`，则此数组将包含证书链中所有证书（从服务器证书到信任根证书）的 `CertificateInfo` 对象。
+
+    否则，它将仅包含服务器证书的 `CertificateInfo` 对象。
+
+- `certificateTransparencyStatus` {{optional_inline}}
+
+  - : `String`。指示连接的[证书透明度](https://certificate.transparency.dev/)状态。可能取值包括：
+
+    - “not_applicable”
+    - “policy_compliant”
+    - “policy_not_enough_scts”
+    - “policy_not_diverse_scts”
+
+- `cipherSuite` {{optional_inline}}
+  - : `String`。用于连接的密码套件，按 [TLS 规范](https://datatracker.ietf.org/doc/html/rfc5246#appendix-A.5)格式表示，如“TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256”。
+- `errorMessage` {{optional_inline}}
+
+  - : `String`。如果 TLS 握手出现问题（例如证书已过期，或找不到受信任的根，或证书被撤销），则 `status` 将为“broken”且 `errorMessage` 属性将包含描述错误的字符串（取自 Firefox 内部错误代码列表）。
+
+    需要注意，你现在只能在 `onHeaderReceived` 监听器中调用 `getSecurityInfo()`，并且在握手失败时不会触发 `onHeaderReceived` 事件。因此在现实中该属性将不会被设置。
+
+- `hsts` {{optional_inline}}
+  - : `Boolean`。如果主机使用[严格传输安全性](/zh-CN/docs/Web/HTTP/Headers/Strict-Transport-Security)则为 `true`，否则为 `false`。
```

Comment:
https://zh.wikipedia.org/wiki/HTTP%E4%B8%A5%E6%A0%BC%E4%BC%A0%E8%BE%93%E5%AE%89%E5%85%A8

```suggestion
  - : `Boolean`。如果主机使用[严格传输安全](/zh-CN/docs/Web/HTTP/Headers/Strict-Transport-Security)则为 `true`，否则为 `false`。
```